core/constants.py

```diff
@@ -121,3 +120,14 @@
 SIMCARD_ASSET_OPTIONS = ((BASIC, "basic"), (BUSINESS, "business"), (NONE, "none"))
 
 TABLES = ["core_history", "django_admin_log"]
+
+NOTIFICATION_TITLES = ()
```

Comment:
What's the essence of this constant?

Reply:
It was supposed to be a tuple containing valid `Notification Titles` for the notifications. title field. I decided against this approach for flexibility sake but didn't remove the variable. I have fixed this though